src/components/templates/Audience/Audience.tsx

```diff
@@ -153,6 +152,22 @@ export const Audience: React.FunctionComponent = () => {
 
   const [iframeUrl, setIframeUrl] = useState<string>("");
 
+  const [hasAlreadyFocussed, setAlreadyFocussed] = useState(false);
```

Comment:
Do we need Already here? Maybe we should shorten this to hasFocus/setFocus?

Reply:
The usecase of this state is to be a flag that is set once, and then never again. Ideally I would have just used a `useEffect` that ran once on page load, but refs aren't guaranteed to be defined at that point, and `useEffects` don't memo over refs, so this was my closest workaround.

You could make it `hasFocussed` or similar if you needed to shorten it, but most of the time I feel that shorter variables for the sake of shortness, when it loses explicitness/context/etc isn't really worthwhile. Particularly when they're going to be transpiled into a couple of random characters in the production assets anyway, so size/etc isn't an issue.